docs/code/getting-started.md

```diff
@@ -111,9 +111,9 @@ Later on, We'll talk about adding other directories like "po" and "data". For no
 ## Gtk.Application {#gtk-application}
 Now what you've been waiting for! We're going to create a window that contains a button. When pressed, the button will display the text "Hello World!" To do this, we're going to use a widget toolkit called GTK+ and the programming language Vala. Before we begin, we highly recommend that you do not copy and paste. Typing each section manually will help you to practice and remember. Let's get started:
 
-1. Create a new file in Scratch and save it as "gtk-hello.vala" inside your "src" folder
+1. Create a new file in Scratch and save it as "Application.vala" inside your "src" folder
```

Comment:
I suppose this is `Create a new file in Code and save it as "Application.vala" inside your "src" folder`, not `in Scratch`.